src/protocol/kademlia/impl/kademlia_impl.cpp

```diff
@@ -635,7 +625,7 @@ namespace libp2p::protocol::kademlia {
     log_.debug("incoming stream with {}",
                stream->remotePeerId().value().toBase58());
 
-    auto session = openSession(stream);
+    auto session = openSession(stream);  // NOLINT(cppcoreguidelines-slicing)
```

Comment:
Please comment why it's not a problem here, such places always are a subject to suspicion.